hugegraph-core/src/main/java/com/baidu/hugegraph/backend/serializer/BytesBuffer.java

```diff
@@ -198,10 +203,26 @@ public BytesBuffer writeString(String val) {
         return this;
     }
 
+    public BytesBuffer writeSortkeys(String val) {
+        this.write(StringEncoding.encode(val));
+        return this;
+    }
+
+    public BytesBuffer writeStringWithEnding(String val) {
+        byte[] bytes = StringEncoding.encode(val);
+        this.write(bytes);
+        this.write(STRING_ENDING_BYTE);
```

Comment:
add comments:
A reasonable ending symbol should be 0x00(to ensure order), but considering that some backends like PG do not support 0x00 string, so choose 0xFF currently.